proc_Hahn_echoph.py

```diff
@@ -10,6 +10,8 @@
 
 
 def select_pathway(s, pathway):
+    if len(pathway) == 0:
+        return s
```

Comment:
this is the easiest solution to the thing that you are noting below:
if the pathway has already been selected, just pass `{}` -- then this doesn't do anything

Reply:
removed function as discussed [here](https://jmfrancklab.slack.com/archives/CV553TPRV/p1724779149443699?thread_ts=1724778674.063839&cid=CV553TPRV)